src/SignalR/server/Core/src/Internal/HubOptionsSetup.cs

```diff
@@ -16,6 +16,8 @@ public class HubOptionsSetup : IConfigureOptions<HubOptions>
 
         internal static TimeSpan DefaultClientTimeoutInterval => TimeSpan.FromSeconds(30);
 
+        internal const int DefaultMaximumMessageSize = 32 * 1024 * 1024;
```

Comment:
Include the default size in the HubOptions.MaximumMessageSize XML doc?

Reply:
Done